### PR TITLE
User api docs

### PR DESCRIPTION
**Pull request by itulkki**
Added documentation for all APIs provided by the user router even though not all of them have been implemented yet.

### CODE REVIEW

**Review by jniemii — APPROVED** [review]

File: docusaurus/docs/api-endpoints/api/user/projects/get-project.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# GET /api/user/projects/:id
+
+Get the project data by id.
+
+## Response
+
+200 on success, 401 if no valid "user-token" cookie is found, and 404 if no project with given id is found. The response body contains the JSON string saved in the database.
```
Comment (hmalmsten) [inline docusaurus/docs/api-endpoints/api/user/projects/get-project.md]: and 404 if current user id does not match projects owner id (same on PUT request)

**Review by hmalmsten — CHANGES_REQUESTED** [review]

File: docusaurus/docs/api-endpoints/api/user/get-settings.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# GET /api/user/settings
+
+Get user settings from the database.
```
Comment (samikuikka) [inline docusaurus/docs/api-endpoints/api/user/get-settings.md:3]: Is there any format for user settings?

**Review by samikuikka — APPROVED** [review]

File: docusaurus/docs/api-endpoints/api/user/projects/get-project.md (hunk 1 above)
Comment (itulkki) [reply docusaurus/docs/api-endpoints/api/user/projects/get-project.md]: this is fixed in a later pull request

File: docusaurus/docs/api-endpoints/api/user/get-settings.md (hunk 2 above)
Comment (itulkki) [reply docusaurus/docs/api-endpoints/api/user/get-settings.md:3]: Added a sentence saying that the backend does not care about the stucture of the JSON.

**Review by samikuikka — APPROVED** [review]

**Review by hmalmsten — APPROVED** [review]

**Review by KorpelainenTommi — APPROVED** [review]

**Pull request merged**